src/components/CalendarMonthOption.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { PropTypes } from 'react';
+import shallowCompare from 'react-addons-shallow-compare';
+import momentPropTypes from 'react-moment-proptypes';
+import moment from 'moment';
+
+export const TOUCHSTART_TIMEOUT = 200;
+
+const propTypes = {
+  month: momentPropTypes.momentObj,
+  modifiers: PropTypes.arrayOf(PropTypes.string),
+  onMonthClick: PropTypes.func,
+  onMonthMouseDown: PropTypes.func,
+  onMonthMouseUp: PropTypes.func,
+  onMonthMouseEnter: PropTypes.func,
+  onMonthMouseLeave: PropTypes.func,
+  onMonthTouchStart: PropTypes.func,
+  onMonthTouchEnd: PropTypes.func,
+  onMonthTouchTap: PropTypes.func,
+};
+
+const defaultProps = {
+  month: moment(),
```

Comment:
this will freeze the default `month` value at the time the module is required. instead, could it either compute `moment()` in the render path, or, have `month` be a thunk that returns a moment object?